plugins/joe_sandbox/plugin.spec.yaml

```
@@ -1,27 +1,42 @@
+---
 plugin_spec_version: v2
 extension: plugin
-products: [insightconnect]
+products:
+  - insightconnect
 name: joe_sandbox
 title: Joe Sandbox
-description: Joe Sandbox Cloud executes files and URLs fully automated in a controlled
-  environment and monitors the behavior of applications and the operating system for
-  suspicious activities
-version: 1.0.4
+description: Joe Sandbox Cloud executes files and URLs fully automated in a
+  controlled environment and monitors the behavior of applications and the
+  operating system for suspicious activities
+version: 2.0.0
+supported_versions: [2.0.0]
```

Regarding the `"Version Increment Validator"`; We have a new field that needs to be included in the `plugin.spec.yaml`, `connection_version`. It should match the major version of the plugin, in this case, `2`. 
```suggestion
supported_versions: [2.0.0]
connection_version: 2
```